### DISCUSSION TITLE
random number generation

### DISCUSSION THREAD

**Discussion by Shammu1966** [community/community#106715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello Sir
I have seen your random number generation mojo based code . copied as it is in my vi test8.mojo file
code is as below
now i am getting error as below
mojo:1:6: error: unable to locate module 'Random'
from Random import rand, randint

my code is as below

from Random import rand, randint
from Pointer import DTypePointer
from DType import DType
from Memory import memset_zero
fn myrandom():
   var p1 = DTypePointer[DType,uint8].alloc(8)
   memset_zero(p1,8)
   print('values at p1:',p1.simd_load[8](0))
   rand[DType,uint8](p1,8)
   print('random values',p1.simd_load[8](0))
   print('only one value :',p1.simd_load[0](0))

fn main():
   myrandom()

**Comment by Shammu1966** [comment]
hello sir a little update
from Random import rand, randint
from Pointer import DTypePointer

the above first two lines were replaced as below

from random import seed
from random import rand, randint
from pointer import DTypePointer

now i am getting error 

now i am getting

**Comment by mw-xyz** [comment]
Should post this in programming help instead, but you did rectify the "R" replaced to "r" as you mentioned, which is good. Could you say which error you're getting? Hope this helps
